src/__tests__/App.test.js

```diff
@@ -3,6 +3,18 @@ import App from '../App';
 
 const { queryByRole, queryByText, queryByTestId } = screen;
 
+// AudioMock
+// Mock out browser Audio API because it doesn't exist in JSDom
+class AudioMock {
+  constructor(filename) {
+    this.filename = filename;
+  }
+  play() {
+    console.log(`Playing sound: ${this.filename}`);
+  }
+}
+window.Audio = AudioMock;
+
```

Comment:
```suggestion
const audio = { play: jest.fn() };
```
Let's pass `audio` in as a prop (defaulted to `new Audio`) instead of attaching a mock class to the window. That'll also make it so that we can spy on `play()` instead of `console.log`, it's more direct.

In `App.svelte`:

```diff
<script>
  import Pomodoro from './Pomodoro.svelte';
  import { formatTime } from './utils';

+ export let audio = new Audio('audio/weird-scream.wav');
```

Reply:
This is much better. Dependency injection > modifying the window global.